app/src/main/java/org/oppia/android/app/profile/ProfileChooserViewModel.kt

```diff
@@ -26,16 +27,73 @@ class ProfileChooserViewModel @Inject constructor(
 ) : ObservableViewModel() {
 
   private val routeToAdminPinListener = fragment as RouteToAdminPinListener
+  private val profileClickListener = fragment as ProfileClickListener
 
+  /** Observable field to track if the add profile button should be shown. */
+  val canAddProfile = ObservableField(true)
+
+  /** Livedata representing the list of profiles on the app, and a model for the 'add' view. */
   val profiles: LiveData<List<ProfileChooserUiModel>> by lazy {
     Transformations.map(
       profileManagementController.getProfiles().toLiveData(), ::processGetProfilesResult
     )
   }
 
+  /** Livedata representing the list of profiles on the app, to be bound to the recyclerview. */
+  val profilesList: LiveData<List<ProfileItemViewModel>> by lazy {
+    Transformations.map(
+      profileManagementController.getProfiles().toLiveData(), ::retrieveProfiles
+    )
+  }
+
+  private fun retrieveProfiles(profilesResult: AsyncResult<List<Profile>>):
+    List<ProfileItemViewModel> {
```

Comment:
```suggestion
  private fun retrieveProfiles(
    profilesResult: AsyncResult<List<Profile>>
  ): List<ProfileItemViewModel> {
```

Prefer this format, instead. We generally use this because it avoids needing to indent the entire method body one further indentation compared to all other same-level members in the class (which makes it stand out).

Reply:
Updated, thanks.